### DISCUSSION TITLE
Node v20 support

### DISCUSSION THREAD

**Discussion by izelnakri** [nanoexpress/nanoexpress#571] [Q&A] [answered]
Hi there, 

I would really like to see a "node.js version guidance" for this package on README since uWebSockets.js always breaks on new edge versions of node. 

As a heavy user of nanoexpress package I cannot plan ahead effectively to see which nodejs version is "recommended" or which version is not supported. I know asking for quick releases is a burden for the maintainer, so what I'm asking is to display prominently on the README.md the "supported nodejs version". I had problems many times in the past on my nodejs package upgrades(and they were important/necessary upgrades for my productivity) due to this packages subdependency(uWebSockets.js), thats why I'm asking this. 

Also, uWebSockets.js now supports node v20, so I would appreciate if we support it as soon a possible but this is not urgent I know support is very new.

**Comment by dalisoft** [comment]
Hi @izelnakri 

Thank you for using nanoexpress when there good alternatives such as **HyperExpress** or **Sifrr** or similar.
I am added **uWebSockets.js v20.24** support but this release does not guarantee stability of update and hope it will work as good as you expect.

Can you please give me example of **Node.js version guidance**? This project follows same Node.js requirement as uWebSockets.js

**Comment by izelnakri** [comment]
@dalisoft I get this error now after updating `nanoexpress` to `v6.2.0`:

```
/home/izelnakri/Github/qunitx/node_modules/fast-query-parse/parse.js:20
  if (str.indexOf('%') !== -1) {
          ^

TypeError: Cannot read properties of undefined (reading 'indexOf')
    at parse (/home/izelnakri/Github/qunitx/node_modules/fast-query-parse/parse.js:20:11)
    at handler (file:///home/izelnakri/Github/qunitx/node_modules/nanoexpress/src/Route.js:431:29)
```

**Comment by dalisoft** [comment, marked as answer]
@izelnakri Locally i'm tested and it now works on **Node v20**
